.pre-commit-config.yaml

```diff
@@ -15,20 +15,9 @@ repos:
     # Ruff version.
     rev: 'v0.6.3'
     hooks:
+      - id: ruff-format
       - id: ruff
         args: ["--fix", "--show-fixes"]
-  # https://github.com/python/black#version-control-integration
-  - repo: https://github.com/psf/black-pre-commit-mirror
-    rev: 24.8.0
-    hooks:
-      - id: black-jupyter
```

Comment:
Is there something similar to Jupyter formatting in ruff as well?

Reply:
I think that can be done using

```toml
[tool.ruff]
# also check notebooks
extend-include = ["*.ipynb"]
```

Reply:
I think it is done by default: https://docs.astral.sh/ruff/configuration/#jupyter-notebook-discovery